src/main/java/com/example/bak/auth/application/command/AuthCommandService.java

```diff
@@ -0,0 +1,44 @@
+package com.example.bak.auth.application.command;
+
+import static com.example.bak.global.exception.ErrorCode.USER_NOT_FOUND;
+
+import com.example.bak.auth.application.command.dto.TokenResult;
+import com.example.bak.auth.application.command.port.AuthCommandPort;
+import com.example.bak.auth.application.command.port.JwtTokenCommandPort;
+import com.example.bak.auth.domain.TokenType;
+import com.example.bak.auth.infra.passwordencoder.PasswordEncoderProvider;
+import com.example.bak.global.exception.BusinessException;
+import com.example.bak.user.domain.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class AuthCommandService {
+
+    private final AuthCommandPort authCommandPort;
+    private final JwtTokenCommandPort jwtTokenCommandPort;
+    private final PasswordEncoderProvider passwordEncoderProvider;
+
+    @Transactional
+    public void registerService(String email, String password) {
+        String encodedPassword = passwordEncoderProvider.encryptPassword(password);
+        authCommandPort.register(email, encodedPassword);
+    }
+
+    @Transactional
+    public TokenResult login(String email, String password) {
+        User user = authCommandPort.findByEmail(email).orElseThrow(() -> new BusinessException(
```

Comment:
```suggestion
        User user = authCommandPort.findByEmail(email)
                        .orElseThrow(() -> new BusinessException())
```

위와 같은 방식으로 줄분리 들어가면 좀 더 예뻐보일 것 같습니다!
메서드 체이닝에 대한 구글 스타일 컨벤션이 있기도 하구요